### PR TITLE
Add munzee netlify

### PR DESCRIPTION
**Pull request by Nightfirecat**
Closes #279

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -4306,6 +4306,18 @@
         ]
     },
 
+    {
+        "name": "Munzee",
+        "url": "https://www.munzee.com/privacy/",
+        "difficulty": "hard",
+        "notes": "Account deletion must be requested via email."
```
Comment (tupaschoal) [inline _data/sites.json]: Missing a comma at the end of the line

File: _data/sites.json [hunk 2]
```diff
@@ -4433,6 +4445,16 @@
         ]
     },
 
+{
```
Comment (tupaschoal) [inline _data/sites.json]: Weird indentation

File: _data/sites.json [hunk 3]
```diff
@@ -4306,6 +4306,18 @@
         ]
     },
 
+    {
+        "name": "Munzee",
+        "url": "https://www.munzee.com/privacy/",
+        "difficulty": "hard",
+        "notes": "Account deletion must be requested via email.",
+        "notes": "Send an e-mail to privacy@freezetag.com. This email should include your first name, last name, e-mail address and, if applicable, your social network ID for the social network from which you access these services. (for example, your Facebook user ID)",
```
Comment (tupaschoal) [inline _data/sites.json]: Notes are duplicated, please keep just one

Comment (Nightfirecat) [reply _data/sites.json]: Didn't even notice that from the original PR... Good catch!

**Comment by tupaschoal** [comment]
Thanks!

**Pull request merged**